sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/DaTypeNameTest.java

```diff
@@ -122,4 +122,33 @@ void testFrom() {
         // then
         assertThat(da2).isEqualTo(da1);
     }
+
+    /**
+     * ListFcEnum = CF, DC, SG, SP, ST , SE
+     * Fc value should be in ListFcEnum to be considered as known
+     */
+    @Test
+    void isUpdatable_ShouldReturnTrue_WhenValImportTrue_And_FCKnown() {
+        // given : nothing
+        DaTypeName da1 = createDa("da1.bda1.bda2", TFCEnum.CF, true, Map.of(0L, "value"));
+        // when then
+        assertThat(da1.isUpdatable()).isTrue();
```

Comment:
if you want it to be more readable , you can write it like this :
```java
// when
boolean updatable = da1.isUpdatable();
// then
assertThat(updatable).isFalse();
```
everywhere

Reply:
I thought that your proposition was less readable that's why i preferred to  regroup when and then in one statement. As introducing a variable only for that seemed to me not a good deal.

Reply:
it's the "Gerkin" syntax, 
but anyway for one assertion maybe you can leave it like that